contributing/releasing.md

```diff
@@ -40,12 +40,16 @@ Run the following command to cut a release:
 
     scripts/release cut
 
+Note: if for some reason this fails, run `scripts/release abort` and try again.
```

Comment:
"First make sure that nobody else is in the middle of cutting a release by visiting the repo and verifying that a release-candidate does not already exist."

Reply:
done